src/less/main.less

```diff
@@ -7,8 +7,13 @@
   transition-timing-function: ease;
 }
 
-.rise-enabled {
+/* do not apply this on the <body> tag */
```

Comment:
Can I ask you why you need to add these lines?

Reply:
one day I was investigating the issue with dark themes; and I found that setting the background on the <body> tag was too intrusive, and so I took it out.

it has no visible impact whatsoever, so I thought I'd keep it as it looked like it could help some day down the road, but if you don't comfortable with that I can take it out

Reply:
> one day I was investigating the issue with dark themes;

Interesting! How does the investigation end?

> and I found that setting the background on the tag was too intrusive, and so I took it out.

Where is applied if it not in the body?

> it has no visible impact whatsoever, so I thought I'd keep it as it looked like it could help some day down the road, but if you don't comfortable with that I can take it out

Thinking about it could help later, and not visible impact... let's keep it!